camayoc/tests/qpc/ui/test_credentials.py

```diff
@@ -64,155 +122,47 @@ def test_create_delete_credential_optional(selenium_browser, qpc_login):
+        .navigate_to(MainMenuPages.CREDENTIALS)
+        .add_credential(credential)
+        .logout()
     )
-    check_auth_type(name, "SSH Key")
-    delete_credential(selenium_browser, {name})
-
 
-@pytest.mark.parametrize("source_type", SOURCE_TYPES)
-def test_edit_credential(selenium_browser, qpc_login, source_type):
-    """Edit a credential's parameters.
 
-    :id: 3a4626ab-c667-41dc-944e-62ac05d51bbe
-    :description: Creates a credential, then attempts to edit all of the
-        parameters that were set, then verifies that they were changed.
-    :steps:
-        1) Log into the UI.
-        2) Go to the credentials page and open the Add Credential modal.
-        3) Fill required + optional fields, using the password option and save.
-        4) Edit the credential parameters and save, make sure it changed.
-    :expectedresults: A new credential is created and then edited.
-    """
-    options = {
-        "name": utils.uuid4(),
-        "username": utils.uuid4(),
-        "password": utils.uuid4(),
-        "source_type": source_type,
-    }
-    if source_type == "Network":
-        options["become_user"] = utils.uuid4()
-    create_credential(selenium_browser, options)
-    new_options = {
-        "name": utils.uuid4(),
-        "username": utils.uuid4(),
-        "password": utils.uuid4(),
-        "source_type": source_type,
-    }
-    if source_type == "Network":
-        new_options["become_user"] = utils.uuid4()
-    edit_credential(selenium_browser, options["name"], new_options)
-    delete_credential(selenium_browser, {new_options["name"]})
-
-
-def test_edit_credential_sshkey(selenium_browser, qpc_login, isolated_filesystem):
-    """Edit the sshkeyfile of a credential.
-
-    :id: 77281d4c-de9f-4db9-9a9f-0c78b0d3811e
-    :description: Creates a credential, then attempts to edit the sshkeyfile
-        to use a new one. The edit is then verified.
-    :steps:
-        1) Log into the UI.
-        2) Go to the credentials page and open the Add Credential modal.
-        3) Fill required fields, using the SSH key option and save.
-        4) Edit the credential parameters and save, make sure it changed.
-    :expectedresults: A new credential is created and then edited.
-    """
-    sshkeyfile = Path(utils.uuid4())
-    sshkeyfile.touch()
-    options = {
-        "name": utils.uuid4(),
-        "username": utils.uuid4(),
-        "sshkeyfile": str(sshkeyfile.resolve()),
-        "source_type": "Network",
-    }
-    create_credential(selenium_browser, options)
-    new_sshkeyfile = Path(utils.uuid4())
-    new_sshkeyfile.touch()
-    new_options = {
-        "sshkeyfile": str(new_sshkeyfile.resolve()),
-        "source_type": "Network",
-    }
-    edit_credential(selenium_browser, options["name"], new_options)
-    delete_credential(selenium_browser, {options["name"]})
+# @pytest.mark.parametrize("source_type", SOURCE_TYPES)
+# def test_edit_credential(ui_client: Client, source_type):
```

Comment:
Wouldn't it be better to mark it as skipped (`@pytest.mark.skip`), instead of commenting it out?

Reply:
I've tried with `skip`, but the code relies on `camayoc/tests/qpc/ui/utils` then it broken into problems. So I thought it was better to comment the code to port it later.